src/core/EdFi.DataManagementService.Core/Middleware/ExtractDocumentInfoMiddleware.cs

```diff
@@ -24,14 +25,22 @@ public async Task Execute(PipelineContext context, Func<Task> next)
             context.FrontendRequest.TraceId
         );
 
-        Debug.Assert(context.FrontendRequest.Body != null, "Body was null, pipeline config invalid");
-        JsonNode documentBody = context.FrontendRequest.Body;
+        if (context.ParsedBody == No.EmptyJsonNode && context.FrontendRequest.Body != null)
+        {
+            var body = JsonNode.Parse(context.FrontendRequest.Body);
```

Comment:
You don't need to do any of this because your middleware already does it.

Reply:
I noticed that without this some test are failing

Reply:
Yes, it's a test setup problem. See my comment on the ExtractDocumentInfoMiddlewareTests